keystore/v2/keystore/importV1.go

```diff
@@ -0,0 +1,140 @@
+		symkey, err := oldKeyStore.ExportSymmetricKey(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export authentication key")
+			return err
+		}
+		defer zeroizeSymmetricKey(symkey)
+		err = s.saveAuthKey(symkey)
+		if err != nil {
+			log.WithError(err).Debug("failed to import authentication key")
+			return err
+		}
+	case filesystemV1.PurposePoisonRecordKeyPair:
+		keypair, err := oldKeyStore.ExportKeyPair(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export poison record key pair")
+			return err
+		}
+		defer zeroizeKeyPair(keypair)
+		err = s.savePoisonKeyPair(keypair)
+		if err != nil {
+			log.WithError(err).Debug("failed to import poison record key pair")
+			return err
+		}
+	case filesystemV1.PurposeStorageClientKeyPair:
+		keypair, err := oldKeyStore.ExportKeyPair(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export client storage key pair")
+			return err
+		}
+		defer zeroizeKeyPair(keypair)
+		err = s.SaveDataEncryptionKeys(key.ID, keypair)
+		if err != nil {
+			log.WithError(err).Debug("failed to import client storage key pair")
+			return err
+		}
+	case filesystemV1.PurposeStorageZoneKeyPair:
+		keypair, err := oldKeyStore.ExportKeyPair(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export zone storage key pair")
+			return err
+		}
+		defer zeroizeKeyPair(keypair)
+		err = s.SaveZoneKeypair(key.ID, keypair)
+		if err != nil {
+			log.WithError(err).Debug("failed to import zone storage key pair")
+			return err
+		}
+	case filesystemV1.PurposeTransportConnectorKeyPair:
+		keypair, err := oldKeyStore.ExportKeyPair(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export AcraConnector transport key pair")
+			return err
+		}
+		defer zeroizeKeyPair(keypair)
+		err = s.SaveConnectorKeypair(key.ID, keypair)
+		if err != nil {
+			log.WithError(err).Debug("failed to import AcraConnector transport key pair")
+			return err
+		}
+	case filesystemV1.PurposeTransportTranslatorKeyPair:
+		keypair, err := oldKeyStore.ExportKeyPair(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export AcraTranslator transport key pair")
+			return err
+		}
+		defer zeroizeKeyPair(keypair)
+		err = s.SaveTranslatorKeypair(key.ID, keypair)
+		if err != nil {
+			log.WithError(err).Debug("failed to import AcraTranslator transport key pair")
+			return err
+		}
+	case filesystemV1.PurposeTransportServerKeyPair:
+		keypair, err := oldKeyStore.ExportKeyPair(key)
+		if err != nil {
+			log.WithError(err).Debug("failed to export AcraServer transport key pair")
+			return err
+		}
+		defer zeroizeKeyPair(keypair)
+		err = s.SaveServerKeypair(key.ID, keypair)
+		if err != nil {
+			log.WithError(err).Debug("failed to import AcraServer transport key pair")
+			return err
+		}
+	default:
+		log.Debug("unknown key purpose")
+		return ErrUnknownPurpose
+	}
+	return nil
+}
+
+func zeroizeSymmetricKey(key []byte) {
+	utils.FillSlice(0, key)
```

Comment:
sometimes we will decide to refactor FillSlice to something like that )) or we will not only zeroize but oneize, secondize too ))

Reply:
Yeah, now that you've mentioned it, this definitely looks like something for `utils`. Should we move these functions there? I remember you've added some in #355:
https://github.com/cossacklabs/acra/pull/355/files#diff-45a2ca38910926a14a2c14c6a04192dd

Reply:
I think will be nice to avoid duplication if sometimes we need it one more time

Reply:
I’ve moved the utilities to `utils`. They will be available for general use once **keystore-v2** lands into master.